GitCommands/Settings/AppSettings.cs

```diff
@@ -1141,6 +1141,12 @@ public static int DiffVerticalRulerPosition
             set => SetInt("diffverticalrulerposition", value);
         }
 
+        public static bool HighlightAllOccurrencesOfSelectedText
```

Comment:
I would prefr myself to not add an option here, to have this the default

Reply:
I am happy to remove the setting to disable this feature.